package.json

```diff
@@ -29,8 +29,8 @@
     "code-push": "node ./scripts/codePush.js"
   },
   "dependencies": {
-    "react": "16.5.0",
-    "react-native": "0.57.0",
+    "react": "16.6.0-alpha.8af6728",
```

Comment:
any particular reason for using alpha version? v.16.6.0 was released 13 days ago?

Reply:
Here: https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#0574

> NOTE WELL: when you upgrade to this version you NEED to upgrade react and react-test-renderer to version "16.6.0-alpha.8af6728" (next version, 0.57.5, will update to 16.6.0, and it will come soon). Also, please check the Known issues section below, especially if you are using Xcode 10.

Reply:
Thanks

Reply:
should not react-test-renderer also be upgraded btw?